### PR TITLE
Unit test for parser interface

### PR DESCRIPTION
**Pull request by j48zheng**

### CODE REVIEW

**Comment by gmichalo** [comment]
I think we should remove the unittest_mysql_dcerrordetector.py from this pull request as this is not final form of this unittest

File: test/Unit_tests/unittest_parser_interface.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+class TestParserInterface(unittest.TestCase):
+
+    def setUp(self):
+        self.session = Session(holo_obj)
+        self.dataset = "../../datasets/unit_test/unit_test_dataset.csv"
+        self.session.load_data(self.dataset)
+
+    def test_load_denial_constraints(self):
+        dcs = self.session.load_denial_constraints(
```
Comment (gmichalo) [inline test/Unit_tests/unittest_parser_interface.py:21]: As we called self.session.load_denial_constraints in every function, we can put it on setUp method

File: test/Unit_tests/unittest_mysql_dcerrordetector.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    def test_number_of_dk_cells(self):
+
+        self.session.load_denial_constraints(
+            "../../datasets/unit_test/unit_test_constraints.txt")
+
+        detector = MysqlDCErrorDetection(self.session.Denial_constraints,
+                                         holo_obj,
+                                         self.session.dataset)
+        self.session.detect_errors(detector)
+        dataframe_C_dk = holo_obj.dataengine.get_table_to_dataframe(
```
Comment (JVonderwell) [inline test/Unit_tests/unittest_mysql_dcerrordetector.py:33]: we can move everything through detect_errors into setup

**Review by JVonderwell — APPROVED** [review]
good once shared code moved into setup

**Comment by ah89** [comment]
Parser interface has a bug that james said will fix it

**Review by thodrek — APPROVED** [review]

**Pull request merged**